pkg/deploy/controller/imagechange/controller_test.go

```diff
@@ -214,20 +233,65 @@ func TestHandle_matchScenarios(t *testing.T) {
 	}
 }
 
-func makeStream(name, tag, dir, image string) *imageapi.ImageStream {
-	return &imageapi.ImageStream{
-		ObjectMeta: kapi.ObjectMeta{Name: name, Namespace: kapi.NamespaceDefault},
-		Status: imageapi.ImageStreamStatus{
-			Tags: map[string]imageapi.TagEventList{
-				tag: {
-					Items: []imageapi.TagEvent{
-						{
-							DockerImageReference: dir,
-							Image:                image,
-						},
-					},
-				},
-			},
+func TestInstantiate(t *testing.T) {
+	tests := []struct {
+		name string
+
+		config         *deployapi.DeploymentConfig
+		noConfigChange bool
```

Comment:
Can you invert all these bools to avoid double-negatives (e.g. noConfigChange -> configChange, nonAutomatic -> automatic)?